adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -422,17 +330,69 @@ class ADAMContextSuite extends ADAMFunSuite {
 
   sparkTest("loadIndexedBam with 1 ReferenceRegion") {
     val refRegion = ReferenceRegion("chr2", 100, 101)
-    val path = resourcePath("sorted.bam")
+    val path = resourcePath("indexed_bams/sorted.bam")
     val reads = sc.loadIndexedBam(path, refRegion)
     assert(reads.rdd.count == 1)
   }
 
   sparkTest("loadIndexedBam with multiple ReferenceRegions") {
     val refRegion1 = ReferenceRegion("chr2", 100, 101)
     val refRegion2 = ReferenceRegion("3", 10, 17)
-    val path = resourcePath("sorted.bam")
+    val path = resourcePath("indexed_bams/sorted.bam")
     val reads = sc.loadIndexedBam(path, Iterable(refRegion1, refRegion2))
     assert(reads.rdd.count == 2)
   }
+
+  sparkTest("loadIndexedBam with multiple ReferenceRegions and indexed bams") {
+    val refRegion1 = ReferenceRegion("chr2", 100, 101)
+    val refRegion2 = ReferenceRegion("3", 10, 17)
+    val path = resourcePath("indexed_bams/sorted.bam").replace(".bam", "*.bam")
+    val reads = sc.loadIndexedBam(path, Iterable(refRegion1, refRegion2))
+    assert(reads.rdd.count == 4)
+  }
+
+  sparkTest("loadIndexedBam with multiple ReferenceRegions and a directory of indexed bams") {
+    val refRegion1 = ReferenceRegion("chr2", 100, 101)
+    val refRegion2 = ReferenceRegion("3", 10, 17)
+    val path = new File(resourcePath("indexed_bams/sorted.bam")).getParent()
+    val reads = sc.loadIndexedBam(path, Iterable(refRegion1, refRegion2))
+    assert(reads.rdd.count == 4)
+  }
+
+  sparkTest("loadBam with a glob") {
+    val path = resourcePath("indexed_bams/sorted.bam").replace(".bam", "*.bam")
+    val reads = sc.loadBam(path)
+    assert(reads.rdd.count == 10)
+  }
+
+  sparkTest("loadBam with a directory") {
+    val path = new File(resourcePath("indexed_bams/sorted.bam")).getParent()
+    val reads = sc.loadBam(path)
+    assert(reads.rdd.count == 10)
+  }
+
+  sparkTest("load vcf with a glob") {
+    val path = resourcePath("bqsr1.vcf").replace("bqsr1", "*")
+
+    val variants = sc.loadVcf(path).toVariantRDD
+    assert(variants.rdd.count === 691)
+  }
+
+  sparkTest("load vcf from a directory") {
+    val path = new File(resourcePath("vcf_dir/1.vcf")).getParent()
+
+    val variants = sc.loadVcf(path).toVariantRDD
+    assert(variants.rdd.count === 681)
+  }
+
+  sparkTest("load parquet with globs") {
+    val inputPath = resourcePath("small.sam")
+    val reads = sc.loadAlignments(inputPath)
+    val outputPath = tempLocation()
```

Comment:
This fails to compile for me on your branch.  Are you depending on a different version of utils?